scripts/vim-patch.sh

```diff
@@ -23,6 +23,7 @@ usage() {
   echo "    -h                 Show this message and exit."
   echo "    -l [git-log opts]  List missing Vim patches."
   echo "    -L [git-log opts]  List missing Vim patches (for scripts)."
+  echo "    -m {vim-revision}  List previous (older) missing Vim patches."
```

Comment:
Note that a lot is missing with the short `"Usage:  ${BASENAME} [-h | -l | -p vim-revision | -r pr-number]"` message above, but duplicating all options there also feels weird?

Reply:
the description doesn't really clarify anything vs `-l` and `-L`. Should it be:

```suggestion
  echo "    -m {rev}  List missing patches older than {rev}."
```

Also is it smart enough to only list patches that touched related files?

Reply:
> Also is it smart enough to only list patches that touched related files?

Not yet. 
￼See https://github.com/neovim/neovim/pull/11514#issuecomment-568259051 and before/around.